.changeset/sharp-garlics-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": patch
```

Comment:
since this is not a bug fix, should this be a minor version update?

Reply:
good call! My bad